src/main/src/main/java/org/geoserver/config/util/XStreamPersister.java

```diff
@@ -658,8 +658,8 @@ void log(Level level, String msg) {
      */
     public void save(Object obj, OutputStream out) throws IOException {
         // unwrap dynamic proxies
-        obj = unwrapProxies(obj);
-        xs.toXML(obj, new OutputStreamWriter(out, "UTF-8"));
+        Object unwrapped = unwrapProxies(obj);
+        xs.toXML(unwrapped, new OutputStreamWriter(out, "UTF-8"));
```

Comment:
This seems functionally the same, any reason for the change? (I don't especially care either way)

Reply:
Debugging with changing objects made me nuts, was hard to compare before and after

Reply:
Thats a good reason, fair enough.